.gitignore

```diff
@@ -4,3 +4,5 @@ build/
 /metrictank
 /cmd/mt-split-metrics-by-ttl/mt-split-metrics-by-ttl
 /cmd/mt-store-cat/mt-store-cat
+/cmd/whisperImporter/whisperReader/main
+/cmd/whisperImporter/chunkWriter/main
```

Comment:
it's just that having a consistent `cmd/mt-foo/main.go` style comes with some advantages. like scripts/build_tools.sh knows how to build all tools by iterating all subdirs.
would it make sense to move the common stuff into the standard package hierarchy? for example `mdata/chunk/archive` could contain your code to create chunk archives